scoresheets/Restaurant.tex

```diff
@@ -1,24 +1,17 @@
-The maximum time for this test is 15 minutes.
+The maximum time for this test is 10 minutes.
 
-\small\begin{scorelist}
+\small\begin{scorelist}[startbutton=false]
 	\scoreheading{Main Goal}
-	\scoreitem{500}{Take and serve an order}
-	\penaltyitem[2]{200}{Being guided to the \emph{Kitchen-bar} or to a customer's table}
-	\penaltyitem{100}{Not making eye-contact when taking an order}
-	\penaltyitem[4]{50}{Bypassing object manipulation (handover)}
-	\penaltyitem[5]{100}{Bypassing object manipulation}
+	\scoreitem[2]{500}{Take and serve an order}
+	\penaltyitem{25}{Request help by receiving directions}
+	\penaltyitem{200}{Request guidance to a table}
+	\penaltyitem[2]{350}{Deliver a partly wrong order}
+	\penaltyitem[2]{500}{Deliver a fully wrong order}
+	\penaltyitem[2]{100}{Handing over an order instead of placing}
 
-	\scoreheading{Second order (bonus reward)}
-	\scoreitem{500}{Take and serve an additional order}
-	\penaltyitem[2]{200}{Being guided to the \emph{Kitchen-bar} or to a customer's table}
-	\penaltyitem{100}{Not making eye-contact when taking an order}
-	\penaltyitem[4]{50}{Bypassing object manipulation (handover)}
-	\penaltyitem[5]{100}{Bypassing object manipulation}
-
-	\scoreheading{Additional bonus rewards}
-	\scoreitem[2]{100}{Detect calling or waving customer}
-	\scoreitem[2]{100}{Reach a customer's table without prior guidance/training}
-	\scoreitem[2]{300}{Use an unattached tray to transport an order}
+	\scoreheading{Bonus Rewards}
+	\scoreitem[2]{250}{Use an unattached tray to transport an order}
```

Comment:
Could we clarify what an "unattached tray" is? I've seen a team putting a basket inside their robot's gripper, does that qualify as unattached? If so, then no one will ever not use that technique and this bonus will always be scored when an order is succeeded (similar to the bonus "leave the arena" in GPSR). =>If this is the intent, then I would just delete any mention of "trays" and give 750 points per order achieved.
The restaurant is a very hard test (only 1 run, unknown location, laymen all around...) and is the emblematic test of @home in my opinion (it's probably the test that has the biggest audience). Therefore, I believe that the points should be tuned in a way that entice most teams to prepare for it.

Reply:
In my opinion it's clear that if a robot starts with the tray or some other container attached to it (even loosely like putting a basket in the opened gripper before starting the test) it can be considered as an attachment to the robot, as it does not require any autonomous interaction by the robot. But I think it could be helpful to define some basic rules on how to achieve the bonus, for instance, would the bonus apply if the barman hands over the tray on request or does the robot need to pick up the tray by its own and if so, where will the tray be placed.

Reply:
What David said. Robot starts with the try in hand or wherever -> tray is considered attached. Robot picks up the tray during the test -> tray is not considered attached.

Reply:
Is the barman allowed to handover the tray of the robot asks?

Reply:
Yes, but I think in order for the robot to score the bonus points it needs to grab the tray itself.